src/maintenance/performance.rst

```diff
@@ -133,11 +133,18 @@ If your system is set up to use the Pluggable Authentication Modules (`PAM`_)
 system (as is the case with nearly all modern Linuxes), increasing this limit
 is straightforward. For example, creating a file named
 ``/etc/security/limits.d/100-couchdb.conf`` with the following contents will
-ensure that CouchDB can open up to 4096 file descriptors at once::
+ensure that CouchDB can open up to 10000 file descriptors at once::
```

Comment:
Just curious: why 10000?

Reply:
Why not? It's just an example.

Further, I see people hitting 4096 in the field fairly often, given max_dbs_open=500 by default, *8 = 4000, and that leaves no fds open for views. Just ~1 view per DB being accessed/built concurrently would get you to 8096 fds without accounting for things like logging.

Reply:
Thanks for the explanation! I miss the new max db open value. Looks pretty fine then.

As for why not. I fear that users will use it blindly for copy-paste-because-docs-said-so-oh-no, so it's better have some sane value here. You just prove that it's sane (: